### ISSUE TITLE
Regular vs. bold line height is not the same

### ISSUE THREAD

**Issue by ddfreyne** [be5invis/Iosevka#355]
* Your font version: **2.3.0**
* Your font variant: **default (Iosevka)**
* Your operating system (name and version): **macOS 10.14.6 (18G84)**
* Your application using Iosevka: **Keynote, TextEdit, … (any Cocoa Mac app?)**
---

The line height in regular Iosevka and bold Iosevka is not the same:

Regular:

![a](https://user-images.githubusercontent.com/6269/65393518-e12f3d80-dd81-11e9-805d-35470accf0d3.png)

With bold:

![b](https://user-images.githubusercontent.com/6269/65393520-e55b5b00-dd81-11e9-9940-d192d30e2da9.png)

Note how the selected word’s selection box is not the same in both cases: it is taller in the bold variant.

(This is especially noticeable when using Iosevka on Keynote slides, where aligning text between slides becomes difficult, as the text moves up/down between slides that have mixed regular/bold text.)

**Comment by ddfreyne** [comment]
This problem does not occur in the preceding release, 2.2.1.

**Comment by be5invis** [comment]
There was a PR changed the line height mechanics. However the numbers are shared across subfamilies so I think that’s an Apple quirk.
cc. @chuling

**Comment by chuling** [comment]
@be5invis Sorry for hearing this, I think it's my fault that I didn't do enough experiments...Maybe we should go back to the original line height mechanics, or just let the new line height mechanics become a optional experimental feature.

**Comment by chuling** [comment]
@be5invis
I think maybe lineGap is blamed for this issue (my fault), take a look at <https://github.com/be5invis/Iosevka/commit/cb4280ed9d4484f3c827b75ec29632671c1d0860>.

@ddfreyne, By setting like this, will this problem still occur?
## meta/aesthetics.ptl

```
set font.hhea.lineGap           (para.leading - asc + DESCENDER)
set font.OS_2.sTypoLineGap      (para.leading - asc + desc)
```

**Comment by be5invis** [comment]
Fixed in 2.3.2. Closed
